tests/main/cloud-init/task.yaml

```diff
@@ -17,11 +17,8 @@ execute: |
     if [[ ! -e /run/cloud-init/instance-data.json ]]; then
         echo "cloud-init instance data is required to execute the test"
 
-        if [[ "$SPREAD_SYSTEM" == ubuntu-* &&
-              "$SPREAD_SYSTEM" != ubuntu-14.04-* &&
-              "$SPREAD_SYSTEM" != ubuntu-core-20-* ]]; then
-            # we expect the test to run on all Ubuntu images,
-            # excluding 14.04/core20
+        if os.query is-ubuntu && not os.query is-trusty; then
```

Comment:
What about != ubuntu-core-20 condition, is it expected to work on core20 now?

Reply:
Ian requested me to remove that because uc20 already support that scenario